testsuites/lucene/src/main/resources/org/eclipse/rdf4j/sail/yeastract_raw.ttl

```diff
@@ -0,0 +1,1436 @@
+
+<urn:raw:yeastract#Yeast_id> <http://www.w3.org/2000/01/rdf-schema#label> "yeast_id" ;
```

Comment:
As discussed: we need to sort out where this data comes from before we can merge this PR in. Just adding a comment to allow me to mark the PR as "needs changes".